claim/models.py

```diff
@@ -83,6 +83,28 @@ def __str__(self):
         return self.name
 
 
+class Layer(models.Model):
+    """
+        Some general representation of map.
+        f.e.
+            All schools of Lviv
+            All universities of Ukraine
+            etc.
+    """
```

Comment:
Idea was to split organizations geographically. One GeoJSON for Kharkiv, another for Lviv. If we are going from file to db, Layer is equal to file. Insted of replacing the file, we change id of Layer in db query. Spliting by types could be done using org_type field. To get all schools of Lviv we could query db to get polygons with org_type = schools, and layer=Lviv